### PR TITLE
Added Redis.md on Databases and fixed multiple broken links on Connec…

### PR DESCRIPTION
**Pull request by shendrekbharath**
Contributors

# Description

Added Redis.md on Databases and fixed multiple broken links on Connectors & Rest api

Fixes #<12>

## Status
**READY**

## Type of change

_Please delete options that are not relevant._

- This change requires a documentation update
- Documentation (fix or adds documentation)

# How Has This Been Tested?

Tested by verifying the correct links

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation (if applicable)

### CODE REVIEW

**Comment by shendrekbharath** [comment]
Hi Fabio,
I have made the changes as you mentioned, please verify. Thank you!

https://github.com/opsdroid/opsdroid/commit/c4bd284a27c9cd98036705199d896ea20430e467

Best Regards,
*Bharath*

File: docs/configuration-reference.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ skills:
     repo: "https://github.com/username/myawesomeskill.git"
 ```
 
-In this configuration we are using the [slack connector](/connectors/slack.md) with a slack [auth token](https://api.slack.com/tokens) supplied, a built-in mongo database connection for persisting data, `hello` and `seen` skills from the official repos and finally a custom skill hosted on GitHub.
+In this configuration we are using the [slack connector](https://github.com/opsdroid/opsdroid/blob/master/docs/connectors/slack.md) with a slack [auth token](https://api.slack.com/tokens) supplied, a built-in mongo database connection for persisting data, `hello` and `seen` skills from the official repos and finally a custom skill hosted on GitHub.
```
Comment (FabioRosado) [inline docs/configuration-reference.md]: Please revert it back to what it was

File: docs/configuration-reference.md [hunk 2]
```diff
@@ -94,13 +94,13 @@ Opsdroid comes with some built-in connectors out of the box. A connector is a mo
...
-- [Rocket.Chat](/connectors/rocketchat.md)
-- [Slack](/connectors/slack.md)
-- [Telegram](/connectors/telegram.md)
-- [Websockets](/connectors/websocket.md)
+- [Facebook](connectors/facebook.md)
+- [GitHub](connectors/github.md)
+- [Matrix](connectors/matrix.md)
+- [Rocket.Chat](connectors/rocketchat.md)
+- [Slack](connectors/slack.md)
+- [Telegram](connectors/telegram.md)
```
Comment (FabioRosado) [inline docs/configuration-reference.md]: Not entirely sure if removing the `/` is the right thing to do here as the links will probably stop working

File: docs/configuration-reference.md [hunk 3]
```diff
@@ -145,8 +145,9 @@ Skills can store data in opsdroid's "memory", this is a dictionary which can be
 
 The built-in databases are:
 
-- [Mongo DB](/databases/mongo.md)
-- [SQLite](/databases/sqlite.md)
+- [Mongo DB](databases/mongo.md)
+- [SQLite](databases/sqlite.md)
```
Comment (FabioRosado) [inline docs/configuration-reference.md]: Same as above, please keep the `/` before databases

File: docs/configuration-reference.md [hunk 4]
```diff
@@ -282,7 +283,7 @@ lang: <ISO 639-1 code -  example: 'en'>
 
 Configure the REST API in opsdroid.
 
-By default, opsdroid will start a web server accessible only to localhost on port `8080` (or `8443` if ssl details are provided). For more information see the [REST API docs](rest-api).
+By default, opsdroid will start a web server accessible only to localhost on port `8080` (or `8443` if ssl details are provided). For more information see the [REST API docs](https://github.com/opsdroid/opsdroid/blob/master/docs/rest-api.md).
```
Comment (FabioRosado) [inline docs/configuration-reference.md]: I have testing this link on my end and it works fine how it was, could you revert it?

**Comment by FabioRosado** [comment]
Hello @shendrekbharath any change of updating this PR in order for me to get it merged? Thank you 👍

**Pull request closed without merging**